config/src/config/netbench.rs

```diff
@@ -30,6 +31,7 @@ impl Default for NetbenchConfig {
             enable_direct_send_testing: false,
             direct_send_data_size: 100 * 1024, // 100 KB
             direct_send_per_second: 1_000,
+            direct_send_ramp_up_ms: Some(30_000), // 30 seconds
```

Comment:
Yeah feels long to me too. I'll play a bit with the injected packet loss, which I suspect might be causing the window size increase to be much slower than it can be.

tcpdump would be able to show the actual window size increase, but it's probably non-trivial to get that on a forge setup. I'm not ready to try that, yet.

Reply:
I'm going to make some changes to this PR to make it more general.

But check out https://www.cs.princeton.edu/courses/archive/fall16/cos561/papers/Cubic08.pdf   Figures 4,5. It takes quite a bit of time to ramp up